index.html

```diff
@@ -1771,6 +1771,50 @@ <h3>Handling changes to worker status</h3>
   <h2>
     Security and privacy considerations
   </h2>
+  <section>
+    <h3>Own contribution estimate</h3>
+    <p>
+      The <dfn>own contribution estimate</dfn> provides a percentage-based indication of how much the current
+      site contributes to the global system pressure. This insight helps developers determine whether
+      a critical or serious pressure state originates primarily from their own application or from
+      external sources.
+    </p>
+    <p>
+      <em>If the majority of the pressure comes from external sources</em>, the site may choose to simplify
+      its features to maintain a smooth experience. It may also prompt users to close other
+      applications to reduce overall system pressure.
+    </p>
+    <p>
+      <em>If the pressure is largely self-inflicted</em>, it indicates that the site's code paths may
+      be too resource-intensive for the device. In such cases, developers might experiment with
+      alternative code paths—for example, offloading calculations from the CPU (via Wasm or
+      JavaScript) to WebGPU—in an effort to enhance performance.
+    </p>
+    <p>
+      We determined that the cost of exposing this additional estimate is negligible. In scenarios
```

Comment:
Was this determination experimental? If so, it'd be helpful to understand how you analyzed the potential for data leakage.

Reply:
This is just from my own experiments with trying to see if I could use the data reliable.